src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -181,12 +236,25 @@ public void WriteCritical(string message, params object[] args)
             }
         }
 
+        /// <summary>
+        /// Writes an exception log.
+        /// </summary>
+        /// <param name="level"><see cref="EventLevel"/></param>
+        /// <param name="innerException"><see cref="Exception"/></param>
```

Comment:
Same comment as one above. This will give users the type information for Exceptions, but we should have info on how the exception is logged.